### DISCUSSION TITLE
Does Next.js 13 server fetch support stale while refresh?

### DISCUSSION THREAD

**Discussion by junaid33** [vercel/swr#2218] [Q&A]
Delba mentioned fetching in Next.js 13 app directory will replace SWR. SWR refetchs when the user leaves and come back to the page. Does Next.js support that out of the box?

**Comment by kylemh** [comment]
Got a link for this tweet you're talking about @junaid33 ?

**Comment by junaid33** [comment]
Delba mentioned it during the Next 13 Conf. swr has this ability to refetch data if the user leaves the webpage or tab and comes back. Next.js 13 app directory handles fetches as a server component and it doesn't refetch when the person leaves the page and comes back.
